MaterialLib/SolidModels/ViscoElasticBurgers-impl.h

```diff
@@ -0,0 +1,332 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef PHYSICALQUANITIES_SOLIDS_VISCOELASTICBURGERS_IMPL_H_
+#define PHYSICALQUANITIES_SOLIDS_VISCOELASTICBURGERS_IMPL_H_
+
+#include <logog/include/logog.hpp>
+
+#include "MaterialLib/SolidModels/KelvinVector.h"
+
+namespace Solids
+{
+template <int DisplacementDim>
+void ViscoElasticBurgers<DisplacementDim>::computeConstitutiveRelation(
+    double const t,
+    ProcessLib::SpatialPosition const& x,
+    double const dt,
+    typename ViscoElasticBurgers<
+        DisplacementDim>::KelvinVector const& /*eps_prev*/,
+    typename ViscoElasticBurgers<DisplacementDim>::KelvinVector const& eps,
+    typename ViscoElasticBurgers<
+        DisplacementDim>::KelvinVector const& /*sigma_prev*/,
+    typename ViscoElasticBurgers<DisplacementDim>::KelvinVector& sigma,
+    typename ViscoElasticBurgers<DisplacementDim>::KelvinMatrix& C,
+    typename MechanicsBase<DisplacementDim>::MaterialStateVariables&
+        material_state_variables)
+{
+    assert(dynamic_cast<MaterialStateVariables*>(&material_state_variables) !=
+           nullptr);
+    MaterialStateVariables& _state =
+        static_cast<MaterialStateVariables&>(material_state_variables);
+    // calculation of deviatoric parts
+    auto const& P_dev = MaterialLib::SolidModels::Invariants<
```

Comment:
suggestion for better readability: `using KelvinInvariants = MaterialLib::SolidModels::Invariants<KelvinVectorSize>;` at the beginning of this function.